### PR TITLE
Add browser tests for most of the wagtail generated organisms

### PR DESCRIPTION
**Pull request by kurtw**
## Additions

- browser tests for most wagtail generated organisms
- `cfgov/scripts/_atomic_helpers.py` to provide streamfield data

## Changes

- initial_test_data script now uses streamfield block data from a new file called `_atomic_helpers.py`.

## Removals

- Post Preview organism from streamfield block choices since it was not used

## Testing

- Run `./cfgov/manage.py migrate`
- run `gulp test:acceptance:browser --specs=wagtail/organisms.js --sauce=false`

## Review

- @richaagarwal 
- @kave 
- @rosskarchner 
- @anselmbradford 
- @sebworks 
- @KimberlyMunoz 
- @jimmynotjim 

## Todos

- More tests (marked with TODO's in code)

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Passes all existing automated tests
* [x] New functions include new tests
* [x] New functions are documented (with a description, list of inputs, and expected output)
* [x] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,14 +18,18 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
...
 
 ### Changes
 - Use bare value of RichText field if value type is not RichText
 - Check against Activity Log topics when generating View More link
+- initial_test_data script now uses streamfield block data from a new file called _atomic_helpers.py.
 
 ### Removed
 - Event RSVP email link button
 - `atomicName` parameter from `checkDom` atomic helper.
+- PostPreview organism since it was unused
```
Comment (jimmynotjim) [inline CHANGELOG.md]: Mind clarifying it's only removed `from streamfield block choices` so no one thinks it's removed from the jinja2 or styles directories.

File: cfgov/scripts/initial_test_data.py [hunk 2]
```diff
@@ -64,55 +67,32 @@ def publish_page(child=None, root=site_root):
...
         contact = Contact(heading='Test User')
     else:
         contact = contact[0]
-    contact.contact_info = StreamValue(contact.contact_info.stream_block, [
-        {"type": "email", "value": {"emails": [{"url": "/", "text": "test@example.com"}]}},
-        {"type": "phone", "value": {"phones": [{"tty": "", "number": "1234567890", "vanity": ""}], "fax": True}},
-        {"type": "address",
-         "value": {"city": "Washington", "title": "", "label": "Address", "state": "DC", "street": "123 abc street",
-                   "zip_code": "20012"}}], True)
+    contact.contact_info = StreamValue(contact.contact_info.stream_block, [atomic.contact_email, atomic.contact_phone, atomic.contact_address], True)
```
Comment (jimmynotjim) [inline cfgov/scripts/initial_test_data.py]: Could this be broken up to avoid going over the 80 char max?

**Comment by richaagarwal** [comment]
Is there a reason we need to run this with the option `--sauce=false`? Just curious what that does. 

Anyway, looks good 👍

**Comment by jimmynotjim** [comment]
@richaagarwal `--sauce=false` just runs the tests locally vs on your saucelabs account.

File: cfgov/v1/atomic_elements/organisms.py [hunk 3]
```diff
@@ -125,13 +111,8 @@ class Meta:
         template = '_includes/organisms/main-contact-info.html'
 
 
-class SidebarContactInfo(blocks.StructBlock):
-    header = blocks.CharBlock(required=False)
```
Comment (kave) [inline cfgov/v1/atomic_elements/organisms.py]: could you verify this doesn't erase content in previously created sidebar contact molecules?

Comment (kurtw) [inline cfgov/v1/atomic_elements/organisms.py]: So, it's not removed just subclassing MainContactInfo which has the same exact fields.

**Comment by kave** [comment]
👍

**Pull request merged**